docs/src/contracts/configurable-constants.md

```diff
@@ -5,7 +5,7 @@ In Sway, you can define `configurable` constants which can be changed during the
 ```rust,ignore
 {{#include ../../../packages/fuels/tests/contracts/configurables/src/main.sw}}
 ```
-Each of the configurable constants will get a dedicated `set` method in the SDK. For example, the constant `STR_4` will get the `set_str_4` method which accepts the same types as defined in sway. Below is an example where we chain several `set` methods and deploy the contract with the new constants.
+Each of the configurable constants will get a dedicated `set` method in the SDK. For example, the constant `STR_4` will get the `set_STR_4` method which accepts the same types as defined in sway. Below is an example where we chain several `set` methods and deploy the contract with the new constants.
```

Comment:
IIRC mention of sway is discouraged since we may support different languages that compile to the vm representation.

```suggestion
Each of the configurable constants will get a dedicated `set` method in the SDK. For example, the constant `STR_4` will get the `set_STR_4` method which accepts the same types as defined in the contract code. Below is an example where we chain several `set` methods and deploy the contract with the new constants.
```